supplementary_style_guide/style_guidelines/general-formatting.adoc

```diff
@@ -329,3 +329,16 @@ See xref:release-notes[Release notes] for guidelines about deprecation notices.
 image:images/no.png[no] We will fix this issue in the 18.3.4 release next February.
 
 image:images/yes.png[yes] It is anticipated that an upcoming release will include a fix for this issue.
+
+[[ip-addresses-and-mac-addresses]]
+== IP addresses and MAC addresses
+
+Use the IP and MAC address ranges that are reserved for documentation purposes to avoid the likelihood of conflicts and confusion:
+
+* Reserved IP address ranges: See the `Using addresses in examples` section in the IBM Style Guide.
```

Comment:
I don't think the other places we reference the ISG in here point to actual section titles. 

Can you change this to something like "See the _IBM Style Guide_ for the list of reserved IP address ranges to use in documentation."

Reply:
I agree with @bergerhoffer here. "Hard-coding" the section name in another guide isn't a good idea because the section name in the ISG could change, and we wouldn't want to have to track those possible changes. 

After this change, this LGTM as well.

Reply:
HI @mmuehlfeldRH . You OK to implement this change? I could give the third approval if it's done by today. I'll be going on PTO On Monday.

Reply:
I updated the reference.

Reply:
Thanks!